guides/v15-mdc-migration.md

```diff
@@ -246,12 +246,12 @@ DOM and CSS of the components, you may need to tweak some of your application's
 
 * Checkbox touch targets are larger, now 40px instead of 16px, which is more accessible. Be sure to
   allow enough space in your layout so that the touch target does not overlap other components. If
-  you are not concerned with accessibility you can match the previous size by using density -5 for
+  you are not concerned with accessibility you can match the previous size by using density -1 for
```

Comment:
Does -1 really match the previous density?

Reply:
In terms of checkbox density in a list, I think that -1 is the closest, whereas in terms on touch target size pretty much all the MDC targets are larger than the legacy version. The `minimum` density is the closest, but it's still 4px larger.